pkg/utils/validator/clusterresourceplacement.go

```diff
@@ -200,6 +203,10 @@ func validateClusterAffinity(clusterAffinity *placementv1beta1.ClusterAffinity,
 func validateTopologySpreadConstraints(topologyConstraints []placementv1beta1.TopologySpreadConstraint) error {
 	allErr := make([]error, 0)
 	for _, tc := range topologyConstraints {
+		if *tc.MaxSkew > 98 {
```

Comment:
where is the number 98 coming from?

Reply:
**reasoning for 98:** there are two buckets for cluster where resource can be scheduled and not scheduled. assuming the distribution is 1:99 or 99:1 the max skew has to be 98.

I spoke with @michaelawyu he suggested there is no point in validating the max skew since it's dynamic as it's based on the number of clusters and it should be done as part of the API validation. @ryanzhang-oss thoughts ?

Reply:
i suggest just checking if it is > 100, and we don't need to do the strict validation here and our implementation should cover it during the runtime.

Reply:
Agree with  zhiying and michael.
Did we put a 100 cluster limit? If not then we don't need to check it here, our implementation can cover it.